glide.go

```diff
@@ -87,6 +88,11 @@ func main() {
 			Usage:  "The temp directory to use. Defaults to systems temp",
 			EnvVar: "GLIDE_TMP",
 		},
+		cli.UintFlag{
+			Name:  "go-get-timeout",
```

Comment:
Please name the parameter in a way that describes what it does. Its provenance is not important in naming.